### ISSUE TITLE
Crash report for Charm version 1.4.1

### ISSUE THREAD

**Issue by MinorFello101** [svenhjol/Charm#192]
Removing optifine did not work and removing charm fixed it.
[crash-2019-11-18_22.10.42-client.txt](https://github.com/svenhjol/Charm/files/3862263/crash-2019-11-18_22.10.42-client.txt)

**Comment by DatrixTHLK** [comment]
Same issue I'm getting without optifine.

Crash Log: https://gist.github.com/DatrixTHLK/bccb164b20050ac39c90eb1ef3d9a885

Minecraft 1.14.4
Forge: 28.1.90
Charm: 1.4.1

**Comment by svenhjol** [comment]
What conditions caused this? Were you changing dimensions?

**Comment by DatrixTHLK** [comment]
Nope. I was launching the game and it crashed for me.

**Comment by svenhjol** [comment]
Both crashlogs have very similar mods on first glance. Are you playing a modpack?

**Comment by MinorFello101** [comment]
It seems that either a forge update or the newest charm update fixed the crash. However, I did remove , add, and update some mods along the way so I can’t be sure. Anyway, for me, it’s “fixed” but I’m just giving a list of removed mods just in case some of the mods I had before were causing it.
Forge: 28.1.94
Minecraft 1.14.4
Charm:1.4.2
Removed mods:
All mobs attack villagers 
Mine and slash
Mine and slash compat
Cyber’s better animal models
Snow real magic
